src/Raven.Client/Documents/Replication/ReplicationNode.cs

```diff
@@ -42,10 +42,10 @@ public string Url
 
         public bool Equals(ReplicationNode other) => IsEqualTo(other);
 
-        public bool IsEqualTo(ReplicationNode other)
+        public virtual bool IsEqualTo(ReplicationNode other)
         {
-            return string.Equals(Url, other.Url, StringComparison.OrdinalIgnoreCase) &&
-                   string.Equals(Database, other.Database, StringComparison.OrdinalIgnoreCase);
+            return string.Equals(Database, other.Database, StringComparison.OrdinalIgnoreCase) && 
```

Comment:
Why did you remove the `Url` comparison?

Reply:
Only for external replication, as I understand that it would be null on the beginning and we set this value later on.

Reply:
Pretty sure that this shouldn't be the case, but for external replication, it uses ConnectionStringName, no? @karmeli87 Can you comment?

Reply:
In external replication we fetch the destination url from the cluster that resides on the 'TopologyDiscoveryUrls', so we can't compare two external replication tasks to be based on the Url.

Reply:
So how do we detect if two external replication are the same or not?

Reply:
http://issues.hibernatingrhinos.com/issue/RavenDB-9171